releasenotes/notes/add-examples-to-characterization-experiments-e77d4d26c6b49694.yaml

```
@@ -0,0 +1,8 @@
+---
+features:
```

```suggestion
other:
```
We put documentation improvements in the `Other` section by convention.

---

OK!

---

OK!